grc/workflows/cpp_hb_nogui/CMakeLists.txt.mako

```diff
@@ -0,0 +1,64 @@
+#####################
+# GNU Radio C++ Flow Graph CMakeLists.txt
+#
+# Title: ${title}
+% if flow_graph.get_option('author'):
+# Author: ${flow_graph.get_option('author')}
+% endif
+% if flow_graph.get_option('description'):
+# Description: ${flow_graph.get_option('description')}
+% endif
+# GNU Radio version: ${config.version}
+#####################
```

Comment:
Question @mbr0wn : Would you prefer to leave this as is,  or not have the exact same block at the beginning of every generated C++, Python, CMake, and potentially a few YAML files; instead, convert to multiline comment and:

```suggestion
#[====[
GNU Radio C++ Flow Graph CMakeLists.txt
<%include file="generated-metadata.inc.mako"/>
]====]
```

and move

```
Title: ${title}
% if flow_graph.get_option('author'):
Author: ${flow_graph.get_option('author')}
% endif
% if flow_graph.get_option('description'):
Description: ${flow_graph.get_option('description')}
% endif
GNU Radio version: ${config.version}
```
to `generated-metadata.inc.mako`.

(we could quite frankly also consider Mako inheritance here, but maybe that's a bridge to far for a PR that just adds a workflow)

Reply:
I would prefer to keep it inline. The complexity of this generated comment is very low, and when I'm reading templates sources I often prefer having much of the text-generation inline. But here we're generating four lines of comments (more if we have a multi-line description) and there's almost no risk of errors in duplicated code, because the output is simply a comment.